configs/recommended.js

```diff
@@ -0,0 +1,95 @@
+const typescriptRules = {
+  'no-unused-vars': 'off',
+  '@typescript-eslint/no-unused-vars': 'off',
+  '@typescript-eslint/no-unused-vars-experimental': ['error', { ignoreArgsIfArgsAfterAreUsed: true }],
+  '@typescript-eslint/no-var-requires': 'off',
+  '@typescript-eslint/no-explicit-any': 'off',
+  '@typescript-eslint/no-use-before-define': 'off',
+  '@typescript-eslint/ban-ts-comment': 'off',
+  'semi': 'off',
+  '@typescript-eslint/semi': 'error',
+  '@typescript-eslint/type-annotation-spacing': 'error',
+  '@typescript-eslint/member-delimiter-style': 'error',
+  '@typescript-eslint/indent': ['error', 2],
+  '@typescript-eslint/explicit-member-accessibility': ['error', { accessibility: 'no-public' }],
+  '@typescript-eslint/explicit-function-return-type': ['error', { allowExpressions: true, allowTypedFunctionExpressions: true }],
+  '@typescript-eslint/explicit-module-boundary-types': ['error', { allowArgumentsExplicitlyTypedAsAny: true }],
+  '@typescript-eslint/ban-types': ['error', { 'types': { '{}': false } }],
+  '@typescript-eslint/no-inferrable-types': ['error', { 'ignoreParameters': false, 'ignoreProperties': true }]
+};
+
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  extends: [
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended'
+  ],
+  plugins: ['react', 'react-hooks', 'import', 'prefer-object-spread', 'siren'],
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module'
+  },
+  env: {
+    browser: true,
+    node: true,
+    es6: true,
+    jest: true
+  },
+  rules: {
+    'no-unused-vars': 'error',
+    'no-console': ['warn', { allow: ['warn', 'error'] }],
+    'eol-last': 'error',
+    'no-multi-spaces': 'error',
+    'import/no-unresolved': ['error', { ignore: ['^ui/.*', '^plugins/.*'] }],
+    'no-multiple-empty-lines': ['error', { max: 2, maxEOF: 0, maxBOF: 0 }],
+    'quotes': ['error', 'single', { 'avoidEscape': true }],
+    'react/no-multi-comp': 'off',
+    'react/display-name': 'off',
+    'react/prop-types': 'off',
+    'react-hooks/rules-of-hooks': 'error',
+    'react-hooks/exhaustive-deps': 'warn',
+    'prefer-object-spread/prefer-object-spread': 'error',
+    'siren/memory-leak': 'warn',
+    'siren/same-core-dependency-version': 'error'
```

Comment:
❤️ this.
Do you think some teams might find it hard/not the top priority to ensure the same version? This might limit our velocity to upgrade dependencies if forcefully enacted. Do you think it should be reduced to 'warn'?

Reply:
I think it's fine to have this as an error as after we manage to bump hapi and friends we should be more aggressive in keeping dependencies up to date to avoid tripping vulnerability scanners or get stuck with old versions.

Reply:
When bumping the `eslint-plugin-siren` version in all the repos ([INVE-12818](https://sirensolutions.atlassian.net/browse/INVE-12818)), my aim was to manually fix the versions where the version differs only in the patch. If there are bigger differences, I'd explicitly override it in the respective repo's `.eslintrc.yml` to `warn` and create a task on the team. Then we have control over how prioritised it is.